api/response/schema.py

```diff
@@ -25,6 +36,16 @@ class Meta:
     response = graphene.Field(ResponseData)
 
 
+class ResponseDetail(graphene.ObjectType):
+    id = graphene.Int()
+    room_id = graphene.Int()
+    created_date = graphene.DateTime()
+    response = graphene.Field(ResponseData)
+    question_type = graphene.String()
+    resolved = graphene.Boolean()
+    state = graphene.String()
```

Comment:
The same Object is in `create_response.py` try and reuse the code to avoid identical blocks

Reply:
Made the requested changes